python/pyarrow/tests/parquet/test_dataset.py

```diff
@@ -1216,7 +1222,6 @@ def test_read_table_duplicate_column_selection(tempdir):
 
 
 def test_dataset_partitioning(tempdir):
-    import pyarrow.dataset as ds
```

Comment:
I don't think this and a few other changes are necessary here, can you revert just those that are formatting-related?